solidity-landing/components/AppFooter.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="app-footer">
+    <app-button
+      v-for="(link, index) in links"
+      :key="index"
+      :text="link.text"
+      :href="link.href"
+      scheme="default"
+      modification="default"
+      size="default"
```

Comment:
why "default", 
`default` value is not set by default for these attributes?

Reply:
Default values in AppButton
```
    scheme: 'flat',
    modification: 'border-circle',
    color: 'primary',
    size: 'medium',
```